src/bridge/cockpitpeer.c

```diff
@@ -183,7 +200,13 @@ on_other_closed (CockpitTransport *transport,
   CockpitPipe *pipe;
   gint status = 0;
 
-  if (self->inited)
+  if (!self->inited)
+    {
+      g_debug ("%s: bridge failed to start%s%s", self->name,
```

Comment:
I think that 'start' covers that case. It could have been run and then failed, or not even be launched, or a helper process that was supposed to launch the bridge failed.